src/styles/utils/_mixins.scss

```
@@ -0,0 +1,4 @@
+@mixin square {
```

```suggestion
@mixin square($size) {
```

Pass argument to mixin and use it

---

Fixed